Content.Shared/Magic/Events/RandomGlobalSpawnSpellEvent.cs

```diff
@@ -0,0 +1,29 @@
+using Content.Shared.Actions;
+using Content.Shared.Storage;
+using System.ComponentModel.DataAnnotations;
+
+namespace Content.Shared.Magic.Events;
+
+public sealed partial class RandomGlobalSpawnSpellEvent : InstantActionEvent, ISpeakSpell
+{
+    /// <summary>
+    /// The list of prototypes this spell can spawn, will select one randomly
+    /// </summary>
+    [DataField]
+    public List<EntitySpawnEntry> Spawns = new();
+
+    /// <summary>
+    /// Whether or not to include players without minds (i.e. disconnected from the server).
+    /// </summary>
+    [DataField]
+    public bool RequireMind = true;
+
+    /// <summary>
+    /// Sound that will play globally when cast
+    /// </summary>
+    [DataField]
+    public string Sound = string.Empty;
```

Comment:
```suggestion
    public SoundSpecifier Sound = new SoundPathSpecifier("/Audio/Magic/staff_animation.ogg");
```
Should be a sound specifier and you can set it as a default.